cluster/engine.go

```diff
@@ -644,8 +650,11 @@ func (e *Engine) RemoveVolume(name string) error {
 
 // RefreshImages refreshes the list of images on the engine.
 func (e *Engine) RefreshImages() error {
+	ctx, cancelFunc := context.WithDeadline(context.Background(), time.Now().Add(time.Minute))
```

Comment:
Let's make a constant for `ImageOperationTimeout` and use it in these functions.